backend-services/README.md

```diff
@@ -106,5 +106,13 @@ be done via
 poetry shell
 ```
 
+## Commands
+
+It's recommended to run this code format command before commiting any changes to
+ensure consistent code style across the project.
+```shell
+poetry run black src tests
+```
```

Comment:
Probably want to also include: `poetry run isort src tests`, since its part of the Python formatting checks.

Reply:
Added :)